source/_integrations/tedee.markdown

```diff
@@ -48,10 +48,76 @@ Local Access Token:
+
+- Tedee PRO
+- Tedee GO
+
 ## Binary sensors
 
-We have three binary sensors: One that indicates whether the battery is currently charging, one indicating if the pull spring is enabled, and one indicating whether the lock is in a "semi-locked" position, meaning the lock has been turned manually and between its normal end positions.
+- **Charging**: indicates whether the battery is currently charging
+- **Pullspring enabled**: is the pull spring setting enabled
+- **Semi locked**: indicates whether the lock is in a "semi-locked" position, meaning the lock has been turned manually and between its normal end positions. The lock itself will be unavailable in this position.
+- **Lock uncalibrated** (disabled by default): Shows when the lock is in an "uncalibrated state"
 
 ## Sensors
 
-The integration currently offers two sensors: A battery sensor, indicating the charge of your lock, and a "pull spring duration" sensor, indicating how long (in seconds) your latch will stay pulled after a pull operation (if supported).
+The integration currently offers two sensors: A **battery** sensor, indicating the charge of your lock, and a **"pull spring duration"** sensor, indicating how long (in seconds) your latch will stay pulled after a pull operation (if supported).
+
+## Possible use-cases
+
+- Auto lock/unlock the door based on presence
+- Control your lock from smart home panels
+- Get alerts when battery is low
+
+## Automations
+
+Get started quickly with these automation examples.
+
+### Automatically lock the door when the last person leaves home
+
+{% details "Example YAML configuration" %}
+
+{% raw %}
+
+```yaml
+alias: Lock door when last person leaves
+description: Lock the door when last person leaves the home
+mode: single
+triggers:
+  - trigger: state
+    entity_id:
+      - zone.home
+    to: "0"
+conditions: []
+actions:
+  - action: lock.lock
+    metadata: {}
+    data: {}
+    target:
+      entity_id: lock.lock_a1b2
+```
+
+{% endraw %} {% enddetails %}
+
+## Known Limitations
+
+This integration only supports functionality that is available locally. This means that 
+
+- Activity logs
+- Updates
+- Key Pads
+
+are not supported.
+
+## Troubleshooting
+
+{% details "Lock state is not updated in real-time" %}
+Make sure your bridge can reach your Home Assistant instance. This means, if you use separate VLANs you need to configure your Firewall appropriately. Additionally, if you have configured an SSL-enabled endpoint for your Internal URL (Settings -> System -> Network -> Home Assistant URL), try setting it back to the IP address of your instance (or a non-HTTPS URL), as HTTPS sometimes leads to problems with the push updates.
+{% enddetails %}
+
+{% details "Authentication failures when trying to use the integration" %}
+The token that is used to talk to your lock is time limited. Sometimes there were issues when the clock of the Home Assistant host was slightly out of sync, so try to sync your host's clock.
```

Comment:
would it make sense to add here how to sync the host's clock? Or is that out of scope?

Reply:
Imo that is out of scope, because it is different depending on what installation method is used/what the host OS is.